spec/sections/copies.bs

```diff
@@ -308,6 +308,13 @@ dictionary GPUImageCopyExternalImage {
         region is copied into the first row of the destination region, and so on.
         The {{GPUImageCopyExternalImage/origin}} option is still relative to the top-left corner
         of the source image, increasing downward.
+
+    Note:
+    It is possible to get blank result when {{WebGL Drawing Buffer|copying from webgl/webgl2 context canvas}}.
```

Comment:
s/blank/a blank/

Reply:
@greggman Sry but I'm not quite ensure how to address this comment. Would you mind to detail a bit? Thanks!

Reply:
The suggestion was "get blank result" should be "get a blank result". The new "get blank results" is also grammatically correct but I think it makes more sense to use the singular "get a blank result"

Reply:
Thanks for explain!